rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -2284,7 +2289,9 @@ error_code cellSaveDataUserFixedDelete(ppu_thread& ppu, u32 userId, PSetList set
 
 void cellSaveDataEnableOverlay(s32 enable)
 {
-	cellSaveData.todo("cellSaveDataEnableOverlay(enable=%d)", enable);
+	cellSaveData.notice("cellSaveDataEnableOverlay(enable=%d)", enable);
+	auto& info = g_fxo->get<save_data_info>();
+	info.enable_overlay = enable != 0;
```

Comment:
atomic_t<bool> has test_and_invert method, it's atomic. For "normal" atomic bool it's not very clear how to do it clearly.